lib/chef/provider/execute.rb

```diff
@@ -39,7 +42,11 @@ def whyrun_supported?
       end
 
       def define_resource_requirements
-         # @todo: this should change to raise in some appropriate major version bump.
+        # @todo: this should change to raise in some appropriate major version bump.
```

Comment:
The comment should be at the right place now since the lines which were added previously are removed now.